docs/1.getting-started/6.data-fetching.md

```diff
@@ -352,3 +352,7 @@ Using `$fetch` has a number of benefits, including:
 It will handle 'smartly' making direct API calls if it's running on the server, or making a client-side call to your API if it's running on the client. (It can also handle calling third-party APIs.)
 
 Plus, it comes with convenience features including automatically parsing responses and stringifying data.
+
+When using server-side rendering, calling `$fetch` *on the client side* without wrapping it with `useAsyncData` causes fetching the data twice initially on the server, then again on the client.
```

Comment:
```suggestion
When using server-side rendering, calling `$fetch`without wrapping it with `useAsyncData` will fetch the data twice - initially on the server, then again on the client.
```

Reply:
Hi @danielroe,

**on the client side** part should be added again because it **only** fetches twice if it is used on the client-side. If it is used on the server side it fetches once. It is so important, I spent hours just trying to understand why `$fetch` in my code executes twice until I find the discussion.

Below is a contrived example which fetches only once using `$fetch`.
```ts
return process.server ? $fetch(...) : useLazyFetch(...);
```

Reply:
Actually the sentence correct by Daniel is correct. Also the example you describe should never exists 😬

Reply:
> example you describe should never exists

@Atinux, I couldn't find a good example at the time. How about this one?

**/server/api/item.ts**
```ts
export default eventHandler(async (event: H3Event) => {
  cont item = $fetch(...);  
  return { item };
});
```

The above example does not cause double fetching. I want to separate the above case from the below case.

**/pages/hello.vue**
```ts
<script setup lang="ts">
  const data = $fetch(...); // double fetching
</script>
```

Could you please suggest a sentence which emphasize the separation between those cases?

Reply:
The data will be fetched at the server side, will *not* be serialized and *not be transferred to the client side and fetched again on the client side.

Reply:
To make it clearer, I rephrased the description and added examples to the docs in a new commit.

@manniL, @Atinux, or @danielroe, could you review whether the examples and descriptions are technically correct? If there are mistakes, please kindly correct them.

If they are technically correct, remember while reviewing the description, that you are part of the Nuxt team and what is very obvious to you may not be so obvious -even confusing- to the developers outside the Nuxt ecosystem.

I try to avoid ambiguity as much as possible. I spent hours why `$fetch` is fetching twice. I saw #18731, just before opening a bug issue. So I think this description is important for others.